THANKS.txt

```diff
@@ -243,6 +243,7 @@ Shashaank N for contributions to scipy.signal.
 Frank Torres for fixing a bug with solve_bvp for large problems.
 Ben West for updating the Gamma distribution documentation.
 Terry Davis for documentation improvements in scipy.ndimage.morphology
+Po-Wen Kao for improving stats.shapiro() API.
```

Comment:
I think the way we are managing the "THANKS.txt" file is that if/when this PR is accepted/merged, an entry would be added to the wiki page: 

https://github.com/scipy/scipy/wiki/*THANKS.txt-additions-modifications-for-1.6.0*

I believe the reason is mostly to avoid merge conflicts, but @rlucas7 will know--the contents of the file may eventually live on a website instead?

Reply:
Thanks. I will remove it in PR and leave it there on wiki page

Reply:
> I think the way we are managing the "THANKS.txt" file is that if/when this PR is accepted/merged, an entry would be added to the wiki page:
> 
> https://github.com/scipy/scipy/wiki/*THANKS.txt-additions-modifications-for-1.6.0*
> 
> I believe the reason is mostly to avoid merge conflicts, but @rlucas7 will know--the contents of the file may eventually live on a website instead?

Yeah as far as I understand the plan is to move thanks to the scipy.org website. The reason is to avoid merge conflicts. Merge conflicts are a pain on maintainers and the release manager-for us though it is more of an inconvenience. For new contributors I think it's an even bigger hassle because of how often the THANKS file creates merge conflicts, then the contributor needs to master `git rebase` or open a new PR if the rebase fails (a common occurrence). 
All this creates a lot more effort that undoubtedly detracts potential new contributors. 

The change to the authors tool was already merged:
https://github.com/scipy/scipy/pull/12793

but the PR to remove the THANKS is still open:
https://github.com/scipy/scipy/pull/12792

I opened the wiki page so that we can move things there as we transition to adding the THANKS on the scipy.org site. 

@tylerjereddy does that clarify?